teambot.py

```diff
@@ -7,17 +7,21 @@
 import atexit
 
 # This directory maps slack channel IDs (not names) to sets of user IDs
-directory = shelve.open('teams.db')
+directory = None
 
 outputs = []
 my_user_id = None
 slack_client = None
 
-def setup(bot):
+def setup(bot, config):
     global my_user_id
     global slack_client
-    slack_client = bot.slack_client
+    global directory
+
+    dir_file = config.get('TEAM_DB_FILE') or 'teams.db'
+    directory = shelve.open(dir_file)
```

Comment:
This name is a little confusing. Something like `db`, `database`, or `dbFile` might be more clear, especially when it's closed later, as in `db.close()`.

Or, maybe it was originally intended to be `dictionary`?